### PR TITLE
docs: update supported kubenetes versions

### PR DESCRIPTION
**Pull request by irespaldiza**

### CODE REVIEW

**Review by RinkiyaKeDad — APPROVED** [review]

File: src/content/enterprise/aks.mdx [hunk 1]
```diff
@@ -28,9 +28,10 @@ This guide will assume that your domain is registered in [AzureDNS](https://docs
 ### Deploy a Kubernetes cluster
 
 We recommend that you follow Azure's [cluster creation guide](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough).
+Okteto supports Kubernetes versions 1.20 through 1.22. Support for 1.19 is deprecated.
```
Comment (rberrelleza) [inline src/content/enterprise/aks.mdx:31]: Remove the deprecated part. I think is clear enough by saying we support X to Y.

**Review by rberrelleza — COMMENTED** [review]
Can we also add it in the latest release notes that we're dropping support for 1.19?

File: src/content/enterprise/aks.mdx (hunk 1 above)
Comment (irespaldiza) [reply src/content/enterprise/aks.mdx:31]: although it is marked for deprecation we support v1.19. From what I understand the change would be:
```
Okteto supports Kubernetes versions 1.19 through 1.22.
```
It is right?

Comment (rberrelleza) [reply src/content/enterprise/aks.mdx:31]: I thought we no longer support 1.19. @maroshii ?

Comment (maroshii) [reply src/content/enterprise/aks.mdx:31]: There's confusion about 1.19 support. We are actually taking it into account in the the 0.15.0 testing plan but marked it as deprecated in the release notes.
+1 for "officially" supporting 1.20 1.21 1.22 only (in the docs) and internally supporting 1.19

**Review by rberrelleza — APPROVED** [review]

**Review by rberrelleza — APPROVED** [review]

File: src/content/enterprise/eks.mdx [hunk 2]
```diff
@@ -28,9 +28,10 @@ This guide will assume that your domain is registered in [Amazon Route53](https:
 ### Deploy a Kubernetes cluster
 
 We recommend that you follow Amazon's [cluster creation guide](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html).
+Okteto supports Kubernetes versions 1.20 through 1.22.
```
Comment (rberrelleza) [inline src/content/enterprise/eks.mdx:31]: @pchico83 do we want to say 1.21 given the issue with volume snapshots?

Comment (pchico83) [reply src/content/enterprise/eks.mdx:31]: @rberrelleza we have tested the rest of the functionality and everything was green. Also, I think it's not about the kubernetes version. It's related to the CSI driver version that you install in the EKS cluster.
Therefore, I would keep 1.22 as supported

**Pull request merged**